documentation/api/commands.markdown

```diff
@@ -69,27 +69,30 @@ processed.
 
 ### "replace catalog", version 6
 
-Version 6 differs from version 5 only in that all field names that were
-previously separated by dashes are separated by underscores.
+* All field names that were previously separated by dashes are
+  separated by underscores.
 
-The payload is expected to be a Puppet catalog, as a JSON object, conforming
-exactly to the [catalog wire format v6][catalogv6]. Extra or missing fields
-are an error.
+* The catalog 'name' field has been renamed to 'certname'.
+
+The payload is expected to be a Puppet catalog, as a JSON object,
+conforming exactly to the [catalog wire format v6][catalogv6]. Extra
+or missing fields are an error.
 
 ### "replace facts", version 4
 
-Similar to version 6 of replace catalog, this version of replace facts differs
-from version 3 only in that the previously dashed fields are now
-underscore-separated.
+* Similar to version 6 of replace catalog, previously dashed fields are now
+  underscore-separated.
+
+* The 'name' field has been renamed to 'certname', for consistency. 
 
 See [fact wire format v4][factsv4] for more information on the
 payload of this command.
 
 ### "deactivate node", version 2
 
-The payload is expected to be the name of a node, as a raw JSON string, which will be deactivated
-effective as of the time the command is *processed*. Serialization of the payload is no
-longer required.
+The payload is expected to be the certname of a node, as a raw JSON
+string, which will be deactivated effective as of the time the command
+is *processed*. Serialization of the payload is no longer required.
```

Comment:
not your change, but I think we should drop the last sentence here. We've dropped references to previous versions in other spots so "no longer" just invites confusion.